internal/cli_utils/datasource_flag.go

```diff
@@ -0,0 +1,31 @@
+package cli_utils
+
+import (
+	"flag"
+	"fmt"
+	"slices"
+)
+
+type datasource_flag struct {
+	Value    string
+	Is_valid bool
+}
+
+func (d *datasource_flag) datasource_binding() {
```

Comment:
func name should be datasouceBinding (if private) and DatasourceBinding (if public)